apps/gpc-verifier/app/hooks/useProofRequest.ts

```diff
@@ -0,0 +1,147 @@
+import { useMemo } from "react";
+import {
+  serializeGPCProofConfig,
+  gpcBindConfig,
+  GPCProofConfig,
+  PODMembershipLists
+} from "@pcd/gpc";
+import { POD_INT_MAX } from "@pcd/pod";
+import { generateSnarkMessageHash } from "@pcd/util";
+
+// https://docs.pcd.team/types/_pcd_gpc.GPCProofConfig.html
+const proofConfig: GPCProofConfig = {
+  // Each POD object mentioned here must be provided in GPCProofInputs.
+  // The names assigned here are used to refer these PODs in GPCProofInputs,
+  // as well as elsewhere in configuration.
+  pods: {
+    govID: {
+      entries: {
+        // Prove the presence of an entry called "idNumber", hide its value,
+        // and also prove that it is not a member of the sanctionsList.
+        idNumber: { isRevealed: false, isNotMemberOf: "sanctionsList" },
+        // Prove the presence of an entry called "firstName", hide its value.
+        firstName: { isRevealed: false },
+        // Prove the presence of an entry called "lastName", hide its value.
+        lastName: { isRevealed: false },
+        // Prove the presence of an entry called "age", hide its value.
+        // and prove that it is >= 18
+        age: {
+          isRevealed: false,
+          inRange: { min: 18n, max: POD_INT_MAX }
+        },
+        // Prove the presence of an entry called "owner", hide its value, and prove
+        // that I own the corresponding Semaphore identity secrets.
+        owner: { isRevealed: false, isOwnerID: true }
+      }
+    },
+    paystub: {
+      entries: {
+        // Prove the presence of an entry called "firstName", hide its value, and
+        // prove that it equals to the firstName in the govID POD.
+        firstName: { isRevealed: false, equalsEntry: "govID.firstName" },
+        // Prove the presence of an entry called "lastName", hide its value, and
+        // prove that it equals to the firstName in the govID POD.
+        lastName: { isRevealed: false, equalsEntry: "govID.lastName" },
+        // There's an entry "currentEmployer" in the paystub POD, because it is
```

Comment:
Interesting demonstration point.

Reply:
I realized there's a mistake in the comment, should be "equals to the lastName in the govID POD", changed.